### DISCUSSION TITLE
part-ii/2016-15/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#340] [Question comments]
Part II 2016 Applied Probability: Paper 1, Section II, J 

https://questions.tripos.org/part-ii/2016-15/

**Comment by ArthurConmy** [comment]
Note: there's a typo in e); it should be $ ||x||^\alpha \vee 1 $ i.e max not min.

<details>
<summary>e)</summary>

For $d=1$, $X$ is always recurrent since the discrete chain is recurrent. The unique invariant measure (up to scaling) is $\pi_x = ||x||^{ - \alpha }$ for $i\neq 0$, and $\pi_0=1$. So by the result that irreducible and non-explosive chains have invariant dist $ 1 / q_x m_x $ iff they're positive recurrent, by IA Analysis we get positive recurrence iff we get convergence of $\sum_{ n \in \mathbb{ Z} } \pi_n $ i.e iff $\alpha > 1$.

For $d=2$ the chain is also recurrent, and with identical invariant distribution by solving detailed balance. Moreover, by partitioning $ \mathbb{ Z}^2 \setminus \{  (0,0) \} $ into $S_1, S_2, ...$ where $S_i = \{ (x,y) : |x|+|y|=i \}$, and noting that $|S_i|=4i$ and $ x \in S_i \implies i^2 / 4 \le ||x||^2 \le i^2$, we get that

$$ \sum_{x \in \mathbb{ Z}^2 \setminus \{  (0,0) \}} \pi_x \propto \sum_{i \ge 1} \frac{4i}{ i^{\alpha} } $$

and this converges iff $\alpha > 2$. For $d \ge 3$ it's not recurrent (I presume we just quote the recurrence parts from IB Markov Chains).

</details>
